### ISSUE TITLE
Evaluation failed: TypeError: Cannot read property 'isWidlike' of undefined

### ISSUE THREAD

**Issue by rodrigooliver** [orkestral/venom#1957]
## Description

When I delete the 'node_modules' folder, it works again for a short time, but then the error reappears.

## Environment

- **Venom version(s):** [4.3.0]
- **Browser:** [Chromium 85]
- **OS:** [Centos 7]
- **Node version:** [Node 16]

## Steps to Reproduce

1. Call function client.sendText() multiple

## Log Output

```
Error: Evaluation failed: TypeError: Cannot read property 'isWidlike' of undefined
    at Object.window.WAPI.window.WAPI.sendCheckType (/home/unitplus/unitplus_node/node_modules/venom-bot/dist/lib/wapi/wapi.js:1:31862)
    at Object.window.WAPI.window.WAPI.sendExist (/home/unitplus/unitplus_node/node_modules/venom-bot/dist/lib/wapi/wapi.js:1:33133)
    at Object.window.WAPI.window.WAPI.sendMessage (/home/unitplus/unitplus_node/node_modules/venom-bot/dist/lib/wapi/wapi.js:1:47248)
    at pptr://__puppeteer_evaluation_script__:2:29
    at ExecutionContext._ExecutionContext_evaluate (/home/unitplus/unitplus_node/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:286:15)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async ExecutionContext.evaluate (/home/unitplus/unitplus_node/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:117:16)
    at async /home/unitplus/unitplus_node/node_modules/venom-bot/dist/api/layers/sender.layer.js:120:28
```

## Your Code

```
try {
      result = await client.sendText(number, message)
      if(result.erro) {
        return false;
      }
      return true;
} catch (e) {
      Sentry.captureException(e);
      return false;
}
```

## Additional context / Screenshot

Usually when I restart the server, it works again, then the error appears again.

**Comment by renandecarlo** [comment]
Same. I'm having this error for months

**Comment by omarmyousef** [comment]
commenting out this line removed the error
tbh i don't know the consequences of doing this, but as far as i know it won't be a big deal
```
if (!Store.WidFactory.isWidlike(e)) return WAPI.scope(e, !0, 404, "The number does not exist. Invalid wid!");
```
it's found at node_modules\venom-bot\dist\lib\wapi\wapi.js:1074

**Comment by orkestral** [comment]
solved #[54d5a7b](https://github.com/orkestral/venom/commit/54d5a7b2cf4aa9523dfe59efde8abed6418d9707)
new version 3.4.1 for update
```javascript
npm i venom-bot@latest
```
